frontend/apps/reader/modules/readerwikipedia.lua

```diff
@@ -17,9 +17,17 @@ function ReaderWikipedia:init()
 end
 
 function ReaderWikipedia:onLookupWikipedia(word, box)
-    -- detect language of the text
-    local ok, lang = pcall(Translator.detect, Translator, word)
-    if not ok then return end
+    -- set language from book properties
+    lang = self.view.document:getProps().language
```

Comment:
The automated build is failing because this patch failed the automated static analysis. You can find the error from our build log: https://travis-ci.org/koreader/koreader/builds/170684711

You should be able to generate the same error using `make static-check` command. This is also documented in `README.md`.

The error here is you need to declare lang as a `local` variable. It's a common practice in Lua community to avoid global variable as much as possible since it causes a big performance penalty and makes the code harder to maintain.